### ISSUE TITLE
Continuing from #2281

### ISSUE THREAD

**Issue by JayQ2K** [joncampbell123/dosbox-x#2288]
As I wrote in #2281 I was able to get a mount done via [config] which makes it show up as a PhysFS drive.
What I was however wondering is that for some files you can just drop them in the root of your DOSBox-X folder and they will be included in the boot process (e.g. MT-32 emulation where the 2 required files can be placed here.

Shouldn't device drivers also be placed in either the root or something like a modules subfolder where you drop all your own stuff into so it can be referred by from DOSBox-X. A subfolder would keep your main installation very clean. That subfolder should then not really need a complete subfolder structure tree beneath it, but moreover be something of a <DROP HERE> folder and fix it in the .conf file.
ULTRASND emulation should definitely not be part of something like this since this is an installation in itself with multiple references to C:\ULTRASND. MT-32 emulation or glide2x.dll or a random device driver (HIMEM) could however be placed here so you'll just need to update the configuration file and you're done.

**Comment by Wengier** [comment]
While it is certainly an interesting idea for auto-inclusion in the boot process, there are a few problems with this approach regarding device drivers. First of all, not all device drivers currently work in DOSBox-X - some may not simply work properly or misfunction, and more work apparently need to be done in this direction. Second, there are many cases you will need to specify parameters for device drivers, but with the auto-inclusion approach as you said these will likely not work. So loading device drivers can not be so automatic, at least for the time being.

**Comment by JayQ2K** [comment]
Yes, am aware of the switches for device drivers. Let's for argument sake
say that anything in the subfolder modules is being made available on the
z-drive (or another drive, requiring the path switch to be updated).

Then I can just do in config:
device = emm386.exe NOEMS.

The MT-32 rom files are made available to the running instance of dosbox in
a very nice way as well. As long as that location is somewhat a 'physical
drive' you are able to work with the switches you set.

And you are indeed still not completely out of the woods concerning device
drivers that don't match well with dosbox, but that problem is not new
since I can make a PhysFS mount on config and still use those drivers.

Op vr 19 feb. 2021 23:43 schreef Wengier <notifications@github.com>:
